### ISSUE TITLE
Broken Dataset Link

### ISSUE THREAD

**Issue by hidaka-ce** [goutamgmb/NTIRE22_BURSTSR#8]
Hello,

Thank you for helding this great challange. 
I noticed that the link to the dataset is broken. Could you please update the link or provide an alternative source for the following dataset?

・Synthetic validation set
    <https://data.vision.ee.ethz.ch/bhatg/synburst_val_2022.zip>

・Synthetic test set 
    https://data.vision.ee.ethz.ch/bhatg/synburst_test_2022.zip

・Real world test set
    https://data.vision.ee.ethz.ch/bhatg/realworld_test_2022.zip

・BurstSR train and validation set
    https://github.com/goutamgmb/NTIRE22_BURSTSR/blob/main/burstsr_links.md

・Zurich RAW to RGB mapping set
    https://data.vision.ee.ethz.ch/bhatg/zurich-raw-to-rgb.zip

Thank you for your help!

Best regards,
